tests/integration_tests/security/test_jail.py

```diff
@@ -104,6 +104,82 @@ def test_empty_jailer_id(test_microvm_with_api):
         assert expected_err in str(err)
 
 
+def test_exec_file(test_microvm_with_api):
+    """
+    Test the jailer option `--exec-file`
+
+    @type: security
+    """
+    test_microvm = test_microvm_with_api
+
+    # Error case 1: No such file exists
+    pseudo_exec_file_path = "/tmp/pseudo_firecracker_exec_file"
+    if os.path.exists(pseudo_exec_file_path):
+        os.remove(pseudo_exec_file_path)
+    test_microvm.jailer = JailerContext(
+        jailer_id="bd65600d-8669-4903-8a14-af88203add38",
+        exec_file=pseudo_exec_file_path,
+    )
+
+    # pylint: disable=W0703
+    try:
+        test_microvm.spawn()
+        # If the exception is not thrown, it means that Jailer run
+        # a non-existing file successfully.
+        assert False
+    except Exception as err:
+        expected_err = (
+            "Jailer error: Failed to canonicalize path {}: "
+            "No such file or directory (os error 2)".format(pseudo_exec_file_path)
+        )
+        assert expected_err in str(err)
+
+    # Error case 2: Not a file
+    pseudo_exec_dir_path = "/tmp/firecracker_test_dir"
+    if not os.path.exists(pseudo_exec_dir_path):
+        os.mkdir(pseudo_exec_dir_path)
+    test_microvm.jailer = JailerContext(
+        jailer_id="bd65600d-8669-4903-8a14-af88203add38",
```

Comment:
I think that this does not matter because the spawned thread fails with invalid `exec_file` input every time.
But this is redundant, so I'll swap this as follows, as @luminitavoicu commented above:
```
test_microvm.jailer.exec_file=pseudo_exec_file_path
```